### PR TITLE
Updates for show/history

### PR DESCRIPTION
**Pull request by allmightyspiff**
#1370 refactored `registration show` to get subnets primarly, changed old `registration show` to `registration history`  since that ismore what the command did. Added a CLI.formatting.trim() function to more easily allow limiting long lines in tables. Fixed a bunch of pylint/flak8 things

### CODE REVIEW

**Review by ATGE — CHANGES_REQUESTED** [review]
### Testing

>  slcli registration -h 
> 
> An unexpected error has occurred:
> .....
> ModuleNotFoundError: No module named 'SoftLayer.CLI.registration.person_history'

**Review by caberos — APPROVED** [review]
good

**Comment by FernandoOjeda** [comment]
Got the same error:

image

**Comment by allmightyspiff** [comment]
@FernandoOjeda / @ATGE Fixed the broken route, everything should work as intended now.

File: SoftLayer/CLI/formatting.py [hunk 1]
```diff
@@ -333,15 +337,24 @@ class FormattedItem(object):
 
         :param original: raw (machine-readable) value
         :param string formatted: human-readable value
+        :param int length: how long the output should be for this item. Trimmed strings end with ...
     """
 
-    def __init__(self, original, formatted=None):
+    def __init__(self, original='NULL', formatted=None, length=None):
```
Comment (ATGE) [inline SoftLayer/CLI/formatting.py:343]: What do you think about a  "None" value instead of "NULL" ?, In my view, looks better as the default value.

**Review by ATGE — COMMENTED** [review]
`slcli registration show`  is working, just I think  a "None" is better than "NULL"
```
slcli registration show
:......:............................................:..........:............:......:..................:...............................:
: Id   : Subnet                                     : Status   : Datacenter : RIR  : Contact          : Notes                         :
:......:............................................:..........:............:......:..................:...............................:
: 966  : 159.122.23.224/27                          : Complete : ams01      : RIPE : Boberson Jackson : Just here for 159.122.23.224  :
: 1973 : 52.116.23.64/28                            : None     : dal13      : ARIN : None             : NULL                          :
: 1064 : 168.1.54.128/28                            : None     : syd01      : RIPE : None             : NULL                          :

```

File: SoftLayer/CLI/formatting.py (hunk 1 above)
Comment (allmightyspiff) [reply SoftLayer/CLI/formatting.py:343]: Thanks for the input, I'm not really sure why I picked Null honestly.

**Comment by allmightyspiff** [comment]
You might need to pull in my latest changes, `person-detail` should be working in the latest changes.

```
$ ./slcli --format=table registration person-detail 1634037
:.......................................:
:            Contact Details            :
:................:......................:
:          Field : Value                :
:................:......................:
: INTERNAL_LABEL : IBM Cloud            :
:  EMAIL_ADDRESS : zzzz@es.ibm.com :
:     FIRST_NAME : ZZZZ:
:      LAST_NAME : Chaparro             :
:        ADDRESS : Avenida Diagonal 571 :
:           CITY : Barcelona            :
:          STATE : OT                   :
:    POSTAL_CODE : 08029                :
:        COUNTRY : ES                   :
:    ABUSE_EMAIL : abuse@ripe.net       :
:          PHONE : 620881761            :
:................:......................:
:......................................................................:
:                          Registered Subnets                          :
:.................:....................:.......................:.......:
:      Subnet     :       Person       :         Status        : Notes :
:.................:....................:.......................:.......:
: 52.116.23.64/28 : Sebastian Chaparro : Registration Complete :   -   :
:.................:....................:.......................:.......:
```

**Pull request merged**